src/intro/objects.md

```diff
@@ -61,18 +61,21 @@ In particular, its responsibilities include:
 A few practical examples:
 
 1. Retrieve a node relative to current -- type inferred as `Gd<Node3D>`:
+
     ```rust
     let child = self.sprite.get_node_as::<Node3D>("Child");
     ```
 
 2. Load a scene and instantiate it as a `RigidBody2D`:
+
     ```rust
     // mob_scene is declared as a field of type Gd<PackedScene>
     self.mob_scene = load("res://Mob.tscn");
     let mut instanced = self.mob_scene.instantiate_as::<RigidBody2D>();
     ```
 
 3. A signal handler for the `body_entered` signal of a `Node3D` in your custom class
+
     ```rust
     #[godot_api]
```

Comment:
Were these empty lines required (CI passed without them before)? Make sure you use the same markdownlint config to avoid unnecessary formatting changes.

Minor thing, but it might turn the list from a "sparse list" into a "spread list", adding `<p>` tags in HTML. Maybe it has no effect though.

Reply:
i don't remember touching this file honestly. might be a leftover of the linting i forgot. one of its settings is in regards to "all code blocks should be spaced". i'm reverting this change anyway.